platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Submission.java

```diff
@@ -10,7 +10,9 @@
  */
 @Entity
 @Table(name = "submission")
-public class Submission extends IdentifiableEntity {
+public class Submission extends IdentifiableEntity implements StoredObject{
```

Comment:
Why would it not make sense? What we store in the database is the identity(implicitly, the path) of the stored object.